src/base/io/writeCbModel.m

```diff
@@ -212,6 +212,9 @@
         end
         %% SBML
     case 'sbml'
+        if (isunix)
+            fileName = strrep(fileName, '~', getenv('HOME'));
```

Comment:
@akaviaLab, it would be safer to do something like this instead
```
if strcmp(fileName, '~') || strncmp(fileName, '~/', 2)
    if ~isempty(getenv('HOME'))
        fileName(1) = [];
        fileName = [getenv('HOME'), fileName];
    end
end
```
as done [here](http://www.mathworks.com/matlabcentral/fileexchange/28249-getfullpath). `strrep` will replace all occurrences of `~` so a `fileName` like `/home/user/tmp/allo~1/test.m` will also get updated.

Could you modify this?

Thanks